docs/console/tutorials/create-extension.md

```diff
@@ -4,204 +4,267 @@ title: Getting started
+
+When you are there, click on the tab _Outbound calls_.
+It will open a new page where you can configure call to external APIs, such as the CRUD collection endpoint of the project connected to Jira.
+
+To configure such call, you need to click on the _Setup outbounds calls_ button. A modal will show up, prompting you to add the following information:
+
+- _Destination URL_: it is the URL that includes the API to call. In this case, it is the URL of the project connected to Jira which depends on how you configured the project (e.g. `https://jira-issues.console.gcp.mia-platform.eu/`)
+- _Authentication required_: if activated, requires authentication to communicate with the API. If the endpoint requires authentication then leave it activated, otherwise you can disable it.
+
+If the authentication is required you will be required to include:
+
+- _Authentication endpoint_: the full URL of the endpoint to send the credentials to authenticate
+- _Authentication Type_: the type of authentication between those available:
+  - _Client ID / Client Secret_: using client credentials, other two fields will be available where you will be prompted to add the _Client ID_ and the _Client Secret_
+  - _Username / Password_: easily understandable, you will also have to include username and password before to create the configuration
+
+Once you selected anything you need to successfully communicate with your external API, you can click to _Save Changes_ and right away see the configuration of the outbound calls updated.
+
+![Extension configured outbound calls](./img/extensions-outbounds-call-detail.png)
+
+You will notice right away that a new URL will be shown in this page that you have not configured by yourself: it just below the label _Base URL to contact your APIs_.
+
+In fact, when you will try to call the API, you will not use the URL of the external project, but a specific Mia-Platform Console API (that will include the `/proxy/extensions/` path)
+that will call used as a dynamic gateway to find the API, handle authentication (if necessary) and execute the requests.
+The URL ends with the `<YOUR_API>` letters, to be replaced with the endpoint you want to connect to (in this case to `/v2/jira-issues`).
+
+### 4. Update the Composer page
+
+Now that we can finally connect to the API, it is finally time to update the Composer page to include the [CRUD Client component](/microfrontend-composer/back-kit/60_components/100_crud_client.md) to allow the table to fetch the data to show.
+
+Go back to the _General_ tab to see to details of the extension and click to the _Edit with Composer_ button.
+From there, you will be redirected to a page where you can see the current configuration of the composer page.
+
+From the `Advanced` section, find the list of the table elements and add the following JSON configuration:
 
 ```json
 {
```

Comment:
I would leave the complete JSON (or at least an idea of where to place the configuration) like:

`
{
   ...
   "iconName": "PiProjectorScreenChartLight",
   "menu": {
    "id": "backoffice-route",
    "labelIntl": {
      "en": "Integrated Backoffice",
      "it": "Backoffice integrato"
    "tag": "bk-crud-client",
    "properties": {
        "basePath": "/proxy/extensions/123456789012345678901234/v2/jira-issues",
        "dataSchema": {
            "$ref": "#/definitions/dataSchema"
        }
    },
    "attributes": {
        "data-mia-label": "CRUD Client"
    },
  "category": ...
  ....
}
`

Reply:
The code removed (including `iconName` and the `menu`) comes from an example that we are not using anylonger and it is the JSON definition on an extension, not the CRUD Client component.

But I like your idea of better explaining where to include the `bk-crud-client` definition, so I'll definitely work on that to find a solution.